starknet_py/net/models/chains.py

```diff
@@ -10,10 +13,33 @@ class StarknetChainId(Enum):
     TESTNET = int_from_bytes(b"SN_GOERLI")
     TESTNET2 = int_from_bytes(b"SN_GOERLI2")
 
+    @classmethod
+    def from_network(cls, net: Network) -> StarknetChainId:
+        """
+        Create a chain from given network.
+        :raises ValueError: when Network is unknown.
+        :param net: Network of the chain.
+        :return: StarknetChainId instance.
+        """
+        net_to_chain = {
+            MAINNET: StarknetChainId.MAINNET,
+            TESTNET: StarknetChainId.TESTNET,
+            TESTNET2: StarknetChainId.TESTNET2,
+        }
+
+        if not isinstance(net, str) or net not in net_to_chain:
+            raise ValueError("Unknown Network.")
```

Comment:
I would change the error message to be more descriptive. When we use it in the Account init and the net is devnet it will raise an error `Unknown Network.` but the user doesn't specify the network parameter in the Account init